packages/cactus-plugin-odap-hermes/src/main/typescript/gateway/client/transfer-initialization.ts

```diff
@@ -35,7 +35,8 @@ export async function sendTransferInitializationRequest(
     sessionData.lastSequenceNumber == undefined ||
     sessionData.sourceGatewayDltSystem == undefined ||
     sessionData.recipientGatewayPubkey == undefined ||
-    sessionData.recipientGatewayDltSystem == undefined
+    sessionData.recipientGatewayDltSystem == undefined ||
```

Comment:
@AndreAugusto11 Adding these additional manadatory input parameters made me think: Is this a breaking API change? Let's say someone deployed the ODAP plugin as it was in `v1.0.0` and now has code running in production that sends requests to it (some business application). And now let's say next week we issue a new release that has this code, they upgrade, try to send their "old" requests without these mandatory parameters and start getting errors. 

Please let me know based on the above example if this is a breaking change or not and then we can take it from there.

Reply:
Hey @petermetz it won't break because this parameter is optional in the plugin options, and if not provided, the list will be initialized as empty.

In the constructor ([line 252 in the new plugin-odap-gateway.ts file](https://github.com/hyperledger/cactus/blob/83c96e727ac1ad15c38b243059667b3d7d2f9233/packages/cactus-plugin-odap-hermes/src/main/typescript/gateway/plugin-odap-gateway.ts#L252).

In the session data ([line 1140 in the new plugin-odap-gateway.ts file](https://github.com/hyperledger/cactus/blob/83c96e727ac1ad15c38b243059667b3d7d2f9233/packages/cactus-plugin-odap-hermes/src/main/typescript/gateway/plugin-odap-gateway.ts#L1140)).

Reply:
@AndreAugusto11 Alright, makes sense.